dvc/objects.py

```diff
@@ -0,0 +1,142 @@
+import logging
+
+from .exceptions import DvcException
+
+logger = logging.getLogger(__name__)
+
+
+class ObjectError(DvcException):
+    pass
+
+
+class ObjectFormatError(ObjectError):
+    pass
+
+
+class HashFile:
+    def __init__(self, path_info, tree, hash_info):
+        self.path_info = path_info
+        self.tree = tree
+        self.hash_info = hash_info
```

Comment:
hash_info for us is like git's object id (aka oid, aka object sha), since, again, we support multiple hashes.